tests/functional/i/invalid/invalid_exceptions/invalid_exceptions_raised.py

```diff
@@ -1,17 +1,13 @@
 # pylint:disable=too-few-public-methods,import-error,missing-docstring, not-callable, import-outside-toplevel
-"""test pb with exceptions and old/new style classes"""
```

Comment:
Let's not remove the test cases.

Reply:
The tests are duplicates and so I removed them.